AutoDD.py

```diff
@@ -21,18 +21,20 @@
 __author__ = "Fufu Fang" "kaito1410"
 __copyright__ = "The GNU General Public License v3.0"
 
-from psaw import PushshiftAPI
-from datetime import datetime, timedelta
-from tabulate import tabulate
-from datetime import datetime
-
+# Native Python imports
 import os
 import math
 import re
 import sys
 import argparse
-from yahooquery import Ticker
+import logging
```

Comment:
Logging is not still used but will be very useful https://docs.python.org/3/library/logging.html